### PR TITLE
Engineering/atmos modsuit changes

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

-Adds emergency tether to the engineerings modsuit.
-Adds magboots to the atmos modsuit.

## Why It's Good For The Game

Tethers are pretty much unused because they're quickly phased out by the jetpack. This adds them to engineering suits as engineering uses them in in EVA at roundstart more often than other groups (notably for solars) so they have at least a little bit of use before they're replaced by jetpacks.

Atmos modsuits get magboots 

## Changelog
:cl: LT3
balance: Pre-equipped engineering modsuit now includes emergency tether
balance: Pre-equipped atmospherics modsuit now includes magboots
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
power creep

**Comment by nikothedude** [comment]
not qol

**Comment by lessthnthree** [comment]
okay

**Review by Fikou — CHANGES_REQUESTED** [review]
atmos techs aren't likely to be doing solar work, they don't really need the tether. i'd rather you give them the magboots, thats been requested a lot

**Comment by lessthnthree** [comment]
> atmos techs aren't likely to be doing solar work, they don't really need the tether. i'd rather you give them the magboots, thats been requested a lot

sure

**Comment by Fikou** [comment]
also deltastation has jetpacks in the engi storage i think that have been largely useless id replace those with more tethers or jetpacks

File: code/modules/mod/mod_types.dm [hunk 1]
```diff
@@ -53,10 +53,13 @@
 		/obj/item/mod/module/welding,
 		/obj/item/mod/module/rad_protection,
 		/obj/item/mod/module/flashlight,
+		/obj/item/mod/module/tether,
 		/obj/item/mod/module/magboot,
 	)
 	default_pins = list(
 		/obj/item/mod/module/magboot,
+		/obj/item/mod/module/flashlight,
```
Comment (Fikou) [inline code/modules/mod/mod_types.dm:61]: go in module order, flashlight would be first
also shouldnt chief engi have it pinned too?

**Comment by ZephyrTFA** [comment]
If I see another balance change marked qol I'm gonna scream into the void more than usual

File: code/modules/mod/mod_types.dm (hunk 1 above)
Comment (lessthnthree) [reply code/modules/mod/mod_types.dm:61]: yeah I'll sort them all to be consistent

**Review by Fikou — APPROVED** [review]

**Comment by Fikou** [comment]
(update changelog)

**Comment by lessthnthree** [comment]
ah yeah

**Comment by Comxy** [comment]
I don't think anyone is going to use those extra tethers, those jetpacks where occationally used as substitution for lack of EVA jetpacks

**Pull request merged**